README.md

```diff
@@ -107,8 +101,149 @@ The `package.json` needs to be in-sync with the [`javascript` track `package.jso
+👁️  C:/Users/Derk-Jan/Documents/GitHub/exercism/javascript-test-runner/test/fixtures/pythagorean-triplet/exemplar/.meta/config.json
+{
+  "blurb": "There exists exactly one Pythagorean triplet for which a + b + c = 1000. Find the product a * b * c.",
+  "authors": ["matthewmorgan"],
+  "contributors": [
+    "ankorGH",
+    "rchavarria",
+    "ryanplusplus",
+    "SleeplessByte",
+    "tejasbubane",
+    "xarxziux"
+  ],
+  "files": {
+    "solution": ["pythagorean-triplet.js"],
+    "test": ["pythagorean-triplet.spec.js"],
+    "example": [".meta/proof.ci.js"]
+  },
+  "source": "Problem 9 at Project Euler",
+  "source_url": "http://projecteuler.net/problem=9"
+}
+
+Enabling tests in C:/Users/Derk-Jan/AppData/Local/Temp/foo-wkg81W/pythagorean-triplet.spec.js
+
+╔═════════════════════════════════════════════════════════════╗
+  🔧 Preparing test project
+╚═════════════════════════════════════════════════════════════╝
+
+✔️  enabling corepack
+✔️  pnpm version now: 9.7.0
+
+✔️  standalone package found
+
+total 2865
+drwxr-xr-x 1 197612 197612      0 Aug  7 23:54 .
+drwxr-xr-x 1 197612 197612      0 Aug  7 23:54 ..
+drwxr-xr-x 1 197612 197612      0 Aug  7 23:54 .docs
+-rw-r--r-- 1 197612 197612    132 Aug  7 23:54 .eslintignore
+-rw-r--r-- 1 197612 197612    318 Aug  7 23:54 .eslintrc
+drwxr-xr-x 1 197612 197612      0 Aug  7 23:54 .meta
+-rw-r--r-- 1 197612 197612     33 Aug  7 23:54 .npmrc
+-rw-r--r-- 1 197612 197612    248 Aug  7 23:54 babel.config.js
+-rw-r--r-- 1 197612 197612    251 Aug  7 23:54 babel.config.js.💥.bak
+-rw-r--r-- 1 197612 197612   2341 Aug  7 23:54 expected_results.json
+drwxr-xr-x 1 197612 197612      0 Aug  7 23:54 node_modules
+-rw-r--r-- 1 197612 197612   2229 Aug  7 23:54 package.json
+-rw-r--r-- 1 197612 197612    859 Aug  7 23:54 package.json.💥.bak
+-rw-r--r-- 1 197612 197612 194571 Aug  7 23:54 pnpm-lock.yaml
+-rw-r--r-- 1 197612 197612    969 Aug  7 23:54 pythagorean-triplet.js
+-rw-r--r-- 1 197612 197612   1865 Aug  7 23:54 pythagorean-triplet.spec.js
+
+Found .pnpm hoisted packages
+total 288
+drwxr-xr-x 1 197612 197612      0 Aug  7 23:54 .
+drwxr-xr-x 1 197612 197612      0 Aug  7 23:54 ..
+-rw-r--r-- 1 197612 197612 194571 Aug  7 23:54 lock.yaml
+
+╔═════════════════════════════════════════════════════════════╗
+  ➤  Execution (tests: does the solution work?)
+╚═════════════════════════════════════════════════════════════╝
+
+✔️  jest tests (*.spec.js) discovered.
+C:\Users\Derk-Jan\AppData\Local\Temp\foo-wkg81W\pythagorean-triplet.spec.js
+
+⚙️  corepack pnpm jest <...>
+
+✅ all tests (*.spec.js) passed.
+
+If the solution previously contained configuration files,
+they were disabled and now need to be restored.
+
+✔️  restoring babel.config.js in output
+✔️  restoring package.json in output
+
+---------------------------------------------------------------
+The results of this run have been written to 'results.json'.
+👁️  C:/Users/Derk-Jan/AppData/Local/Temp/foo-wkg81W/results.json
```

Comment:
This all looks like a lot of console print. Also your info in it. Was this intended?

Reply:
Yes